content/en/digital_certificate.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Digital Certificate
```

Comment:
Why are we calling this a digital certificate instead of just certificate? I don't necessarily object but the title was less clear to me than it should be. @iamNoah1 @seokho-son @CathPag I think this item demonstrates why we need to add a metadata field for alternative names. An alternate name section would let us provide other common names for a particular entry.